addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,168 @@
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the Onkyo Receiver
+                </description>
+            </parameter>
+            <parameter name="port" type="integer" required="true"
+                min="1" max="65335">
+                <label>Onkyo Receiver Port</label>
+                <description>Port of the Onkyo to control</description>
+                <default>60128</default>
+            </parameter>            
+        </config-description>
+    </thing-type>
+
+
+    <!-- Commands -->
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Power on/off your device</description>
+    </channel-type>
+    <channel-type id="input">
+        <item-type>Number</item-type>
+        <label>Input source</label>
+        <description>Select the input source of the AVR</description>
+        <state>
+            <options>
+                <option value="00">DVR/VCR</option>
+                <option value="01">SATELLITE/CABLE</option>
+                <option value="02">GAME</option>
+                <option value="03">AUX</option>
+                <option value="04">GAME</option>
+                <option value="05">PC</option>
+                <option value="16">BLURAY/DVD</option>
+                <option value="32">TAPE1</option>
+                <option value="33">TAPE2</option>
+                <option value="34">PHONO</option>
+                <option value="35">CD</option>
+                <option value="36">FM</option>
+                <option value="37">AM</option>
+                <option value="38">TUNER</option>
+                <option value="39">MUSICSERVER</option>
+                <option value="40">INTERETRADIO</option>
+                <option value="41">USB</option>
+                <option value="42">USB_BACK</option>
+                <option value="43">NETWORK</option>
+                <option value="45">AIRPLAY</option>
+                <option value="48">MULTICH</option>
+                <option value="50">SIRIUS</option>
+            </options>
+        </state>
+        
+    </channel-type>    
+    <channel-type id="mute">
+        <item-type>Switch</item-type>
+        <label>Mute</label>
+        <description>Mute/unmute your device</description>
+    </channel-type>
+    <channel-type id="volume">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>
+        <description>Volume of your device</description>
+        <state min="0" max="100" step="1" pattern="%d %%">
+        </state>
+    </channel-type>
+    <channel-type id="next">
+        <item-type>Switch</item-type>
+        <label>Next</label>
+        <description>Send a next command to the player</description>
+    </channel-type>
+    <channel-type id="prev">
```

Comment:
I'd echo @kaikreuzer's comments.  I just had a play with the UI and these channels have the same outcome as the previous/next control channels so they're seem redundant.